src/memory_manager.rs

```diff
@@ -31,8 +30,12 @@ use std::sync::atomic::Ordering;
 /// Arguments:
 /// * `mmtk`: A reference to an MMTk instance.
 /// * `tls`: The thread that will be used as the GC controller.
```

Comment:
These arguments have changed

Reply:
Yes. I need to update the comments as well.